metals/src/main/scala/scala/meta/internal/metals/watcher/ProjectFileWatcher.scala

```diff
@@ -172,17 +177,21 @@ object ProjectFileWatcher {
           watcher.register(root, Int.MaxValue)
         }
 
-        watcher
+        new FileWatcher {
+          override def start(): Unit = {}
+          override def cancel(): Unit = watcher.close
+        }
       } else {
         // Other OSes register all the files and directories individually
-        val watcher = initWatcher(watchFilter, watchEventQueue)
-
-        pathsToWatch.directories.foreach(
-          watcher.register(_, Int.MaxValue)
-        )
-        pathsToWatch.files.foreach(watcher.register(_, -1))
-
-        watcher
+        new FileWatcher {
+          override def start(): Unit = {}
+          override def cancel(): Unit = SimpleFileWatcher.initWatcher(
```

Comment:
I may be missing something, but this looks like it should be the implementation for `start`, not `cancel`

Reply:
Oh yeah, you're totally right, don't know what I was thinking there. :sweat_smile: